devnet/vitosha-3/currencies.json

```diff
@@ -212,75 +232,171 @@
                     "network": "JUNO",
                     "ch": "channel-190"
                 }
+            },
+            {
+                "a": {
+                    "network": "NOLUS",
+                    "ch": "channel-116"
+                },
+                "b": {
+                    "network": "NEUTRON",
+                    "ch": "channel-209"
+                }
+            },
+            {
+                "a": {
+                    "network": "NEUTRON",
+                    "ch": "channel-8"
+                },
+                "b": {
+                    "network": "AXELAR",
+                    "ch": "channel-???"
+                }
+            },
+            {
+                "a": {
+                    "network": "NEUTRON",
+                    "ch": "channel-1"
+                },
+                "b": {
+                    "network": "COSMOS_HUB",
+                    "ch": "channel-???"
+                }
             }
         ]
     },
-    "lease": {
-        "Lpn": ["USDC"],
-        "Lease": {
-            "ATOM": {
-                "swap_routes": [
-                    [
-                        {
-                            "pool_id": "12",
-                            "pool_token": "OSMO"
-                        },
-                        {
-                            "pool_id": "5",
-                            "pool_token": "USDC"
-                        }
+    "protocols": {
+        "osmosis_axlusdc": {
+            "DexNetwork": "OSMOSIS",
+            "Lpn": [
+                {
+                    "USDC": {
+                        "currency@dex": "USDC_AXELAR"
+                    }
+                }
+            ],
+            "Lease": {
+                "ATOM": {
+                    "currency@dex": "ATOM",
```

Comment:
```suggestion
                    "remote_currency": "ATOM",
```

Reply:
`remote` has too broad meaning, each network != Nolus is remote.

Reply:
Then perhaps `dex_currency`? I'd like to avoid special symbols as it would just cause problems when trying to de-/serialize and glue attributes to even make it work.